src/deck.js

```diff
@@ -0,0 +1,26 @@
+const cards = require( './card' )
+const { Card, SUITS, NUMBERS } = cards
+
+class Deck {
+  constructor() {
+    this.cards = this.createDeck()
+  }
+
+  createDeck() {
+    return SUITS.reduce( this.deckReducer, [] )
+  }
+
+  deckReducer( previous, current, index, array ) {
+    previous.concat( NUMBERS.reduce( ( previousCard, currentCard ) => {
+      previous.push( new Card({ number: currentCard, suit: current }))
+    }, [] ))
+
```

Comment:
Does previous.concat combine the previous cards to the new card to create the full deck of cards? Will we be using the same for the game logic? - Harman